src/components/[guild]/activity/ActivityLogFiltersBar/ActivityLogFiltersBar.tsx

```diff
@@ -1,27 +1,86 @@
-import { GridItem, SimpleGrid } from "@chakra-ui/react"
-import StickyBar from "components/common/Layout/StickyBar"
+import { GridItem, SimpleGrid, useColorModeValue } from "@chakra-ui/react"
 import useGuild from "components/[guild]/hooks/useGuild"
+import StickyBar from "components/common/Layout/StickyBar"
+import RadioButtonGroup from "components/common/RadioButtonGroup"
 import { useActivityLog } from "../ActivityLogContext"
+import { ActivityLogActionGroup } from "../constants"
+import { useActivityLogFilters } from "./components/ActivityLogFiltersContext"
 import DateRangeInput from "./components/DateRangeInput"
 import FiltersInput from "./components/FiltersInput"
 
+const options = [
+  {
+    label: ActivityLogActionGroup.UserAction,
+    value: ActivityLogActionGroup.UserAction.toString(),
+  },
+  {
+    label: ActivityLogActionGroup.AdminAction,
+    value: ActivityLogActionGroup.AdminAction,
+  },
+]
+
 const ActivityLogFiltersBar = (): JSX.Element => {
-  const { isUserActivityLog } = useActivityLog()
+  const { isUserActivityLog, withActionGroups, actionGroup, setActionGroup } =
+    useActivityLog()
   const { featureFlags } = useGuild()
 
   const shouldShowDateRangeInput = isUserActivityLog || featureFlags?.includes("CRM")
 
+  const { clearFilters } = useActivityLogFilters()
+
+  const btnActiveColor = useColorModeValue(
+    "var(--chakra-colors-indigo-100)",
+    "var(--chakra-colors-indigo-500)"
+  )
+
+  const changeActionGroup = (value: ActivityLogActionGroup) => {
+    setActionGroup(value)
+    clearFilters(["action"])
+  }
+
+  const ActionGroupButtons = (): JSX.Element => (
+    <GridItem colSpan={{ base: 3, md: 2 }}>
+      <RadioButtonGroup
+        options={options}
+        radioGroupProps={{
+          onChange: (newValue) =>
+            changeActionGroup(newValue as ActivityLogActionGroup),
+          value: actionGroup,
+          defaultValue: ActivityLogActionGroup.UserAction,
+        }}
+        buttonGroupStyleProps={{ w: { base: "full", md: "auto" }, spacing: 1.5 }}
+      ></RadioButtonGroup>
+    </GridItem>
+  )
+
   return (
     <StickyBar>
       <SimpleGrid columns={3} gap={4}>
-        <GridItem colSpan={shouldShowDateRangeInput ? { base: 3, md: 2 } : 3}>
-          <FiltersInput />
-        </GridItem>
-
-        {shouldShowDateRangeInput && (
-          <GridItem colSpan={{ base: 3, md: 1 }}>
-            <DateRangeInput />
-          </GridItem>
+        {withActionGroups ? (
```

Comment:
This is an acceptable solution, but I'm curious if we could make a "smarter" grid here by using the `Grid` component [with `gridTemplateAreas`](https://chakra-ui.com/docs/components/grid#template-areas) instead of a `SimpleGrid`? 🔥 
Could you please experiment with it a little bit?